scripts/build.js

```diff
@@ -104,8 +104,8 @@ const text = await response.text();
 const parsed = parse(text);
 const functions = generateFunctions(parsed);
 
-fs.writeFileSync(
```

Comment:
Synchronous writing is significantly faster than async when you don't need concurrency, so the only way this change is beneficial is if you do the two file writings with `Promise.all`.

Reply:
Reverted.